presto-orc/src/main/java/com/facebook/presto/orc/reader/SliceDictionaryStreamReader.java

```diff
@@ -262,28 +289,40 @@ private void openRowGroup(Type type)
     private static void readDictionary(
```

Comment:
It is confusing which arguments are input and which are output.  Maybe add a comment like `Reads dictionary into data and offsetVector`